server/modules/monitoring/DeploymentMonitor.js

```diff
@@ -7,12 +7,14 @@ let fs = require('fs');
 let co = require('co');
 let ms = require('ms');
 let logger = require('../logger');
+let moment = require('moment');
```

Comment:
When we add new modules to a file, could we opt for `const` until it's shown that the value needs to be mutable.

Reply:
Yes I will change that now.